cookbook/config/save_agent.py

```diff
@@ -0,0 +1,37 @@
+from agno.agent.agent import Agent
+from agno.db.postgres import PostgresDb
+from agno.models.openai import OpenAIChat
+from agno.tools.duckduckgo import DuckDuckGoTools
+
+db = PostgresDb(db_url="postgresql+psycopg://ai:ai@localhost:5532/ai")
+
+agent = Agent(
+    id="agno-agent",
+    model=OpenAIChat(id="gpt-4o-mini"),
+    name="Agno Agent",
+    version="v1",
+    db=db,
+    tools=[DuckDuckGoTools()],
+)
+
+agent.print_response("How many people live in Canada?")
+
+# Save the agent to the database
+agent.save()
```

Comment:
As much as I love this, it becomes dodgy with AgentOS if users save their agents to the agent's DB. So safer would be to just have `save(agent, db)` to force users to specify a shared DB. Of course they don't have to, but it would fit better with the whole flow I think